app/services/google/scrape_service.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Google
+  class ScrapeService
+    attr_reader :keyword
+
+    def initialize(keyword_id:)
+      @keyword_id = keyword_id
+    end
+
+    # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/AbcSize
+    def call
+      keyword = Keyword.find(@keyword_id)
```

Comment:
if we defined `attr_reader :keyword_id` above, here we just use `keyword_id` without the `@`

Reply:
Fixed in https://github.com/Goose97/google-scraper-ruby/pull/20/commits/b0713248018bbc87ba507fb4a5ffa388a807a3df#diff-c3acbe2de6be98fe66d47a5ec8d1965cf566a1e37807230b81b87e88c875cb27L14-L15